src/Drupal/ContentTrait.php

```
@@ -231,6 +232,59 @@ public function contentChangeModerationStateWithTitle(string $content_type, stri
+    $nid = end($nids);
+    $node = Node::load($nid);
+
+    // @codeCoverageIgnoreStart
+    if (!$node instanceof NodeInterface) {
+      throw new \RuntimeException(sprintf('Unable to find "%s" content with title "%s".', $content_type, $title));
+    }
+    // @codeCoverageIgnoreEnd
+    $handler = \Drupal::entityTypeManager()->getAccessControlHandler('node');
+    assert($handler instanceof NodeAccessControlHandlerInterface);
+    $handler->acquireGrants($node);
+  }
```

PHPMD (2.15.0)

[error] The parameter $content_type is not named in camelCase. (undefined)

(CamelCaseParameterName)